edison/config.py

```diff
@@ -0,0 +1,28 @@
+import secrets
+import importlib, inspect
+import sys, inspect
+
+config_dict = {}
+
+# If config_dict is empty this function builds it dynamically 
+# and returns the appropriate config object path.
+def get_config_object(env_keyword: str):
+    if(len(config_dict) == 0):
+        # Iterating through all config.py members
+        for name, obj in inspect.getmembers(sys.modules[__name__]):
+            # We're interested only with the derived classes of the Config class
```

Comment:
you're not really checking here is the class you've got is derived from Config, only that is is not the `Config` class itself

Reply:
ack